code/modules/cargo/packs.dm

```diff
@@ -1430,6 +1430,15 @@
 	crate_name = "robotics assembly crate"
 	crate_type = /obj/structure/closet/crate/secure/science
 
+/datum/supply_pack/science/aiupload
+	name = "AI Upload Board"
```

Comment:
```suggestion
	name = "AI Upload board crate"
```

Reply:
er, the capitalization for this suggested change looks a bit weird